qemu/tests/timedrift_check_after_load_vm.py

```diff
@@ -0,0 +1,161 @@
+import re
+import time
+import logging
+
+from avocado.utils import process
+from virttest import env_process
+from virttest import error_context
+from virttest import guest_agent
+from virttest import arch
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Clock check after savevm and loadvm
+    Note: Run this test case, make sure that guest agent is installed
+    and start inside the guest.
+
+    1. On host, load kvm module with "kvmclock_periodic_sync=N"
+    2. On host, sync time with ntp server
+    2. Boot guest with qemu guest agent
+    3. On guest, sync time with ntp server
+    4. On guest, stop chronyd
+    5. Run samevm and loadvm
+    6. Run qemu-guest-agent command: guest-set-time
+    7. Run qmp command: rtc-reset-reinjection
+    8. On guest, query the time offset with ntp server,
+       it should be less than 3 seconds
+
+    :param test: Kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    def _load_kvm_module_with_kvmclock_periodic_sync(module_param):
+        """
+        Load kvm module with kvmclock_periodic_sync=N/Y
+
+       :params module_param: the value of kvmclock_periodic_sync
+       """
+        error_context.context("_load_kvm_module_with_kvmclock_periodic_sync=%s"
+                              % module_param, logging.info)
+        check_modules = arch.get_kvm_module_list()
+        error_context.context("check_module: '%s'" % check_modules, logging.info)
+        check_modules.reverse()
+        for module in check_modules:
+            rm_mod_cmd = "modprobe -r %s" % module
+            process.system(rm_mod_cmd, shell=True)
+        check_modules.reverse()
+        for module in check_modules:
+            load_mod_cmd = "modprobe %s" % module
+            if module == "kvm":
+                load_mod_cmd = "%s kvmclock_periodic_sync=%s" % (load_mod_cmd, module_param)
+            process.system(load_mod_cmd, shell=True)
+        check_mod_cmd = params["check_mod_cmd"]
```

Comment:
For ppc64 and ppc64el, this case will error here for no parameter "check_mod_cmd". Is that acceptable for now?